scripts/pi-hole/php/savesettings.php

```diff
@@ -137,12 +137,17 @@ function readAdlists()
 	{
 		while (($line = fgets($handle)) !== false)
 		{
-			if(substr($line, 0, 5) === "#http")
+			if(substr($line, 0, 1) === "#")
 			{
-				// Commented list
-				array_push($list, [false,rtrim(substr($line, 1))]);
+				// Comments start either with "##" or "# "
+				if(substr($line, 1, 1) !== "#" &&
+			           substr($line, 1, 1) !== " ")
```

Comment:
These comparisons can be simplified by using `$line[1]`

Reply:
What is `$line[1]` does not exist, i.e., the line is only
```
#
```
?